### PR TITLE
ARP: Rawns.Reddit-Wallpaper-Changer

### PR DESCRIPTION
**Pull request by ghost**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19251)

### CODE REVIEW

File: manifests/r/Rawns/Reddit-Wallpaper-Changer/1.0.13/Rawns.Reddit-Wallpaper-Changer.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 PackageIdentifier: Rawns.Reddit-Wallpaper-Changer
-Publisher: Rawns
-PackageName: Reddit-Wallpaper-Changer
+Publisher: Paul Rawnsley
+PackageName: Reddit Wallpaper Changer
 Author: Rawns
 ShortDescription: Reddit Wallpaper Changer is a lightweight C# application for Windows that will scrape Reddit for fresh desktop wallpapers. You can specify which subs to scrape from and how oftern to rotate your wallpaper.
```
Comment (ryfu-msft) [inline manifests/r/Rawns/Reddit-Wallpaper-Changer/1.0.13/Rawns.Reddit-Wallpaper-Changer.yaml:5]: small typo, "oftern" should be "often"

**Review by ryfu-msft — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]
Hello bot? 🤔

**Comment by ItzLevvie** [comment]
cc @ryfu-msft: you might have to remove the "Policy-Test-2.3" label for the PR to merge.

**Comment by OfficialEsco** [comment]
> cc @ryfu-msft: you might have to remove the "Policy-Test-2.3" label from the PR so it can be merged.

Its actually the Changes Required on the wrong line that is making it not pass

**Review by ryfu-msft — APPROVED** [review]

**Pull request merged**